tardis/apps/publication_workflow/components/PublicationsList.jsx

```diff
@@ -0,0 +1,148 @@
+import PropTypes from 'prop-types';
+import React, { Fragment, useEffect, useState } from 'react';
+import { Modal, Button } from 'react-bootstrap';
+import PublicationCard from './PublicationCard';
+import { deletePub, retractPub } from './utils/FetchData';
+import PublicationToast from './utils/PublicationToast';
+
+
+const PublicationsList = ({
+  releasedPubsList,
+  retractedPubsList,
+  scheduledPubsList,
+  draftPubsList,
+  onPubUpdate,
+  onResumeDraft,
+  listViewType,
+}) => {
+  const [deleteModalOpen, setDeleteModalOpen] = useState(false);
+  const [retractModalOpen, setRetractModalOpen] = useState(false);
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
    const [retractModalOpen, setRetractModalOpen] = useState(false);
```